### PR TITLE
Fixed the system navbar behaviour on theme change

### PR DESCRIPTION
**Pull request by k12onoss**
I made the system navigation bar (and status bar) transparent and set the SystemUiMode property to edgeToEdge so as to keep up with the latest android designing guidelines. It also fixes the problem of the status bar elements being hidden in some cases.

### CODE REVIEW

**Comment by Sangwan5688** [comment]
When you learnt flutter? Anyways, some screens aren't inside safe area, coz of which navigation bar will show over the app UI. Can you please fix those, also please dart format the files too. Some aren't formatted, which is breaking the build.

**Comment by k12onoss** [comment]
Learnt it over the last year :).

> navigation bar will show over the app UI

Yes, that is the intended outcome. Most system apps in Android (and all apps on iOS) have the navigation bar floating over the app UI. I think it looks much better in comparison.

Anyways, I have reformatted the code, so it shouldn't fail now.

**Comment by Sangwan5688** [comment]
I get that, but some parts of the app will be inaccessible on some screens. Let's take miniplayer for example. The screens which are inside safearea, those will work just fine, but for screens which aren't inside safe area the miniplayer will show below the navigation bar. It's not an issue with the gesture navigation bar, but with 3 buttons, miniplayer won't be accessible at all.

**Comment by k12onoss** [comment]
Ahh, okay. Wrapped the miniplayer and madeBy text within the SafeArea widget. Let me know if there are any other components that should be wrapped.

File: lib/main.dart [hunk 1]
```diff
@@ -257,74 +257,81 @@ class _MyAppState extends State<MyApp> {
   Widget initialFuntion() {
     return Hive.box('settings').get('userId') != null
         ? HomePage()
-        : AuthScreen();
+        : AuthScreen();g
```
Comment (Sangwan5688) [inline lib/main.dart:260]: umm?

**Review by Sangwan5688 — CHANGES_REQUESTED** [review]

File: lib/main.dart (hunk 1 above)
Comment (Sangwan5688) [inline lib/main.dart:260]:
```suggestion
        : AuthScreen();
```

**Pull request merged**